store/localstore/mvcc_test.go

```diff
@@ -301,3 +301,13 @@ func (t *testMvccSuite) TestBufferedIterator(c *C) {
 	c.Assert(it.Value(), DeepEquals, []byte("2"))
 	tx.Commit()
 }
+
+func encodeInt(n int) []byte {
```

Comment:
These functions was in `kv_test.go` and used by both `kv_test.go` and "mvcc_test.go" before. As `localsotre/kv_test.go` was moved to `store/store_test.go` in this commit, we need add them back to `mvcc_test.go`.